Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.ShowCase.ts2

```diff
@@ -0,0 +1,22 @@
+/**
+* NodeTypes around Show Cases
+*/
+
+
+prototype(Neos.NeosIo:Reference.ShowCase.Document) < prototype(Neos.NeosIo:Reference.Document) {
+	body {
+		content {
+			main = Neos.NeosIo:Reference.ShowCase
```

Comment:
No problem, I adjusted the bug, so it should at least have everything correct now, unstlyed, but without bugs, hopefully.